molgenis-data-csv/src/main/java/org/molgenis/data/csv/CsvRepositoryCollection.java

```diff
@@ -29,7 +30,7 @@
  */
 public class CsvRepositoryCollection extends FileRepositoryCollection
 {
-	public static final String NAME = "CSV";
+	public static final String NAME = RepositoryCollection.BACKEND.CSV.name();
```

Comment:
CSV doesn't really feel like a backend.
Plus isn't there possibly a one to many relation between RepositoryCollection class and actual backend instance of the RepositoryCollection?